app/Navigation/NavigationMetaFields.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Navigation;
+
+class NavigationMetaFields {
+
+	/**
+	 * NavigationMetaFields constructor.
+	 */
+	public function __construct() {
+		add_action( 'wp_nav_menu_item_custom_fields', [ self::class, 'addCustomFields' ], 10, 2 );
+		add_action( 'wp_update_nav_menu_item', [ self::class, 'updateNavItems' ], 10, 2 );
+	}
+
+	/**
+	 * Mapping of the meta fields to the menu item.
+	 *
+	 * @data-structure: 'FIELD_DESCRIPTION' => [
+	 *    'meta_field' => 'META_FIELD_NAME',
+	 *    'text' 	   => 'TEXT_SHOWN_BESIDE_CHECKBOX'
+	 * ]
+	 *
+	 * @return array[]
+	 */
+	public static function getFields() {
```

Comment:
Mierenneuken:
```
public static function getFields(): array
{
    //
}
```

Reply:
Dan ga ik er van uit dat we in dit project alle functies strict typing maken? (Even voor de zekerheid)

Reply:
Wat mij betreft wel! Echter ben ik maar 1 persoon natuurlijk.